adapters/handlers/rest/configure_api.go

```diff
@@ -750,12 +751,15 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 			}
 		}, appState.Logger)
 	}
-	enterrors.GoWrapper(
-		func() {
-			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-			defer cancel()
-			backupScheduler.CleanupUnfinishedBackups(ctx)
-		}, appState.Logger)
+	if !entcfg.Enabled("DISABLE_CLEANUP_UNFINISHED_BACKUPS") {
```

Comment:
This won't work... you have to change it to `entcfg.Enabled(os.Getenv("DISABLE_CLEANUP_UNFINISHED_BACKUPS"))`, you can checkout other places where we use this helper